docs/app.md

```diff
@@ -4,29 +4,34 @@ An app installed in an organization from a repository.
-| `description`     | `String`                           | App description, from the repository. E.g. `"Manage an organization’s token supply and distribution."`.                  |
-| `htmlPath`        | `String`                           | The path of the app HTML page. Stays empty if the app doesn’t have a frontend. E.g. `/index.html`                        |
-| `htmlUrl`         | `String`                           | The HTTP URL of the app HTML page. Uses the IPFS HTTP provider. E.g. `http://gateway.ipfs.io/ipfs/QmdLEDDfi…/index.html` |
-| `contentUrl`      | `String`                           | The HTTP URL of the app content. Uses the IPFS HTTP provider. E.g. `http://gateway.ipfs.io/ipfs/QmdLEDDfi…/`             |
-| `icons`           | `{ src: String, sizes: String }[]` | Array of icons for the app (follows the web app manifest `icons` format).                                                |
-| `name`            | `String`                           | Name of the app, from the repository. E.g. `"Tokens"`.                                                                   |
-| `registryAddress` | `String`                           | Address of the aragonPM registry for this app.                                                                           |
-| `registry`        | `String`                           | Name of the aragonPM registry for this app. E.g. `"aragonpm.eth"`                                                        |
-| `sourceUrl`       | `String`                           | URL of the app source code.                                                                                              |
-| `version`         | `String`                           | The current version of the app.                                                                                          |
-| `kernelAddress`   | `String`                           | The address of the kernel.                                                                                               |
-| `isForwarder`     | `Boolean`                          | Whether the app can act as a forwarder.                                                                                  |
-| `tags`            | `String[]`                         | Tags associated with the app.                                                                                            |
+| Name                | Type                               | Description                                                                                                              |
+| ------------------- | ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `abi`               | `Abi`                              | The Ethereum contract ABI of the app contract.                                                                           |
```

Comment:
Note that `abi`, `intents`, and `deprecatedIntents` don't need to be methods as we already have that information available.